tyk-headless/templates/deployment-gw-repset.yaml

```diff
@@ -63,6 +63,8 @@ spec:
         env:
           - name: TYK_GW_LISTENPORT
             value: "{{ .Values.gateway.containerPort }}"
+          - name: TYK_GW_CONTROLAPIPORT
+            value: "{{ .Values.gateway.control.containerPort }}"
```

Comment:
we also need to add this to the `ports` of the container

```yaml
        ports:
        - containerPort: {{ .Values.gateway.control.containerPort }}
   ```

Reply:
added it! (i think)